packages/plugin-aws-sdk/package.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "opentelemetry-plugin-aws-sdk",
-    "version": "0.0.13",
+    "version": "0.1.0-alpha.0",
```

Comment:
I guess this should be changed to a non-alpha release?

Reply:
It will be changed by lerna from CI when released